### ISSUE TITLE
script_mvel_rce (elastic search) exploit doesn't work

### ISSUE THREAD

**Issue by ahm3dhany** [rapid7/metasploitable3#99]
I'm running metasploitable3 on virtualbox, kali-linux on vmware and the host os is windows 7.

## Host System
* OS: Windows 7
* Packer Version: 0.12.3
* Vagrant Version:  1.9.2
* VirtualBox Version: 5.1.14

Although the "script_mvel_rce" module says "The target is vulnerable" after executing "check" command, it gave me "[*] Exploit completed, but no session was created." after executing "run" or "exploit".

```
msf exploit(script_mvel_rce) > check
[+] 172.28.128.3:9200 The target is vulnerable.
msf exploit(script_mvel_rce) > run

[*] Started bind handler
[*] Trying to execute arbitrary Java...
[*] Discovering remote OS...
[+] Remote OS is 'Windows Server 2008 R2'
[*] Discovering TEMP path
[+] TEMP path identified: 'C:\Windows\TEMP\'
[!] This exploit may require manual cleanup of 'C:\Windows\TEMP\Oef.jar' on the target
[*] Exploit completed, but no session was created.
```

the target machine runs "Elasticsearch REST API 1.1.1" on port 9200:

```
msf exploit(script_mvel_rce) > services

Services
========

host          port   proto  name          state  info
----          ----   -----  ----          -----  ----
...
172.28.128.3  9200   tcp    http          open   Elasticsearch REST API 1.1.1 name: Jessica Jones; Lucene 4.7
...
```

I tried to:
[1] reboot both kali & metasploitable3 machines

[2] switch the firewall off from both kali & metasploitable3

 but the module still gives me the same result:

"[*] Exploit completed, but no session was created."

**Comment by evilgash** [comment]
Which payload did you specify? I had problems with java/meterpreter/reverse_https

`[*] Meterpreter session 1 opened (192.168.56.110:8443 -> 192.168.56.103:49297) at 2017-03-03 20:55:49 +0100`
`[*] Exploit completed, but no session was created.`

However,  java/meterpreter/reverse_tcp worked fine

**Comment by ahm3dhany** [comment]
@evilgash 
Actually all the java/meterpreter/* payloads didn't work for me..
But after changing the connections of the target virtual machine (i.e. metasploitable3) & the attacker virtual machine(i.e. kali) to Bridged and set the payload to java/shell_reverse_tcp (or java/shell/reverse_tcp).. I managed to get a shell session (not a meterpreter session).

I tried to upgrade the shell session to meterpreter session using "post/multi/manage/shell_to_meterpreter" module but unfortunately didn't work.
